shared/dataflow/codeql/dataflow/internal/DataFlowImpl.qll

```diff
@@ -4284,11 +4259,19 @@ module MakeImpl<LocationSig Location, InputSig<Location> Lang> {
       /** Holds if this node is a source. */
       final predicate isSource() { super.isSource(_) }
 
-      /** Holds if this node is a grouping of source nodes. */
-      final predicate isSourceGroup(string group) { this = TPathNodeSourceGroup(group) }
+      /**
+       * DEPRECATED: This functionality is no longer available.
+       *
+       * Holds if this node is a grouping of source nodes.
+       */
+      deprecated final predicate isSourceGroup(string group) { none() }
 
-      /** Holds if this node is a grouping of sink nodes. */
-      final predicate isSinkGroup(string group) { this = TPathNodeSinkGroup(group) }
+      /**
+       * DEPRECATED: This functionality is no longer available.
+       *
+       * Holds if this node is a grouping of sink nodes.
+       */
+      deprecated final predicate isSinkGroup(string group) { none() }
```

Comment:
So to be clear, we're deprecating _and_ removing the feature, the only thing that's still supported is that existing code will still compile (but ignore souce/sink grouping) thanks to these stubs, for the deprecation cycle.

Reply:
Yes.